compareECLFiles.cmake

```diff
@@ -95,6 +95,38 @@ function(add_test_compareECLFiles)
                         TESTNAME ${PARAM_CASENAME})
 endfunction()
 
+function(add_test_compareNewECLFiles)
```

Comment:
i think a better name should be found, what does 'new' mean?

Reply:
Here, new means that this is not a regression test, i.e. this test compares two "new" runs and not an old result file with a new run.

Reply:
So maybe 'compareSeparate'.

Reply:
Ok :+1: